lib/sigstore/signer.rb

```diff
@@ -25,8 +26,11 @@ class Signer
     include Loggable
 
     def initialize(jwt:, trusted_root:)
-      @jwt = jwt
+      @identity_token = OIDC::IdentityToken.new(jwt)
       @trusted_root = trusted_root
+
+      @verifier = Verifier.for_trust_root(rekor_url: @trusted_root.tlogs_for_signing.first.base_url,
```

Comment:
is finding the first valid tlog in the trusted root sufficient?

Reply:
Yep -- signing only requires one source of signed time/transparency, so picking the first valid tlog is correct. 

(Technically you can pick any valid tlog, but AFAIK the public instance only lists one anyways.)